Sources/Build/Command.link(SwiftModule).swift

```diff
@@ -84,6 +93,11 @@ extension Command {
         case .Library(.Dynamic):
             args.append("-emit-library")
         case .Executable:
+            // FIXME: This is inefficient, we can store the main module in enum payload.
```

Comment:
Can we just fix this? I care less about the efficiency than it would just make the code more clearly correct that there is always exactly one executable module for an executable link.